services/settings/pkg/store/defaults/defaults.go

```diff
@@ -24,6 +24,29 @@ const (
 	SettingUUIDProfileDisableNotifications = "33ffb5d6-cd07-4dc0-afb0-84f7559ae438"
 	// SettingUUIDProfileAutoAcceptShares is the hardcoded setting UUID for the disable notifications setting
 	SettingUUIDProfileAutoAcceptShares = "ec3ed4a3-3946-4efc-8f9f-76d38b12d3a9"
+
+	// SettingUUIDProfileEmailSendingInterval is the hardcoded setting UUID for the email sending interval setting
+	SettingUUIDProfileEmailSendingInterval = "08dec2fe-3f97-42a9-9d1b-500855e92f25"
+	// SettingUUIDProfileEventShareCreated it the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventShareCreated = "872d8ef6-6f2a-42ab-af7d-f53cc81d7046"
+	// SettingUUIDProfileEventShareRemoved is the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventShareRemoved = "d7484394-8321-4c84-9677-741ba71e1f80"
+	// SettingUUIDProfileEventShareExpired is the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventShareExpired = "e1aa0b7c-1b0f-4072-9325-c643c89fee4e"
+	// SettingUUIDProfileEventSpaceShared is the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventSpaceShared = "694d5ee1-a41c-448c-8d14-396b95d2a918"
+	// SettingUUIDProfileEventSpaceUnshared is the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventSpaceUnshared = "26c20e0e-98df-4483-8a77-759b3a766af0"
+	// SettingUUIDProfileEventSpaceMembershipExpired is the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventSpaceMembershipExpired = "7275921e-b737-4074-ba91-3c2983be3edd"
+	// SettingUUIDProfileEventSpaceDisabled is the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventSpaceDisabled = "eb5c716e-03be-42c6-9ed1-1105d24e109f"
+	// SettingUUIDProfileEventSpaceDeleted is the hardcoded setting UUID for the send in app setting
+	SettingUUIDProfileEventSpaceDeleted = "094ceca9-5a00-40ba-bb1a-bbc7bccd39ee"
+	// SettingUUIDProfileEventPostprocessingStepFinished is the hardcoded setting UUID for the send in mail setting
+	SettingUUIDProfileEventPostprocessingStepFinished = "fe0a3011-d886-49c8-b797-33d02fa426ef"
+	// SettingUUIDProfileEventScienceMeshInviteTokenGenerated is the hardcoded setting UUID for the send in mail setting
+	SettingUUIDProfileEventScienceMeshInviteTokenGenerated = "b441ffb1-f5ee-4733-a08f-48d03f6e7f22"
```

Comment:
Just discussed with @bastianbeier:

We don't need a setting to switch of ScienceMeshInvitationToken emails. It should always be active as it makes no sense to turn that notification of.